pkg/jaeger/query/trace_query.go

```diff
@@ -108,6 +109,14 @@ const (
 	`
 )
 
+var (
+	statusCodeTagValueToDBValue = map[string]string{
+		"OK":    "ok",
```

Comment:
I bet otel defines these so let's maybe use defined values from the library? We can uppercase/downcase if needed as well.